gilbreth_support/config/conveyor_objects.yaml

```diff
@@ -1,11 +1,49 @@
-objects:
-  topics: [gear_description, piston_rod_description, pulley_description, gasket_description, disk_description]
-  initial_pose:
-    position: [1.2, 4.0, 1.5]
-    orientation: [0.0, 0.0, 0.0]
-  reference_frame: world
-  lateral_placement_variance: 0.2   # [m] Distance range for objects to be placed transversely on the conveyor
-  yaw_placement_variance: 180.0     # [deg] Rotation angle range for objects to be placed on the conveyor
-  spawn_timing_variance: 1.0        # [s] Delay
+spawner:
+  reference_frame: world            # Gazebo frame in which to spawn the parts
   spawn_period: 8.0                 # [s] Time between component spawnings
   randomization_seed: 0             # Seed for randomization engine
+  objects:
+    - name: gear                        # Name of the spawned object
+      mesh_resource: package://gilbreth_support/meshes/conveyor_objects/visual/gear.stl
+      initial_pose:                     # Initial object pose relative to the reference frame
+        position: [1.2, 4.0, 1.0]
+        orientation: [0.0, 0.0, 0.0]
+      lateral_placement_variance: 0.2   # [m] Distance range for objects to be placed transversely on the conveyor
+      yaw_placement_variance: 180.0     # [deg] Rotation angle range for objects to be placed on the conveyor
+      spawn_timing_variance: 1.0        # [s] Delay
```

Comment:
Is this ```spawn_timing_variance``` parameter applied as a range on top of the spawner node's ```spawn_period``` time value?

Reply:
Yes; this is documented in a README in #18